developer/ro-crate-api.md

```diff
@@ -2,9 +2,7 @@
 title: RO-Crate Submission API
 ---
```

Comment:
I was volunteered to add those.. will have a go now that I can edit through this PR.

Reply:
I was wondering if we could not just change the URLs in workflow hub, since this will be a lot of work. Could we check on google how many of these deep links exist on google?